src/components/Modal/index.tsx

```diff
@@ -16,13 +16,25 @@ export type ModalProps = {
   onToggle?: () => void;
   /**
    * Whether or not the top section with the close button should be absolute or not.
-   * If yes. It won't push the content down and the padding should be set on the children component.
+   * If yes. It won't push the content down and the hasPadding should be set on the children component.
```

Comment:
`hasPadding` does not exist anymore. I think this comment should remain `If yes. It won't push the content down and the padding should be set on the children component.`? I am not sure, please just make it clear according to the actual behaviour after your changes were made.